lib/chef/provider/aws_rds_instance.rb

```diff
@@ -36,14 +36,12 @@ def destroy_aws_object(instance)
     end
     # Wait up to 10 minutes for the db instance to shutdown
     converge_by "waited until RDS instance #{new_resource.name} was deleted" do
-      wait_for(
-        aws_object: instance,
-        query_method: :exists?,
-        expected_responses: [false],
-        acceptable_errors: [AWS::RDS::Errors::DBInstanceNotFound],
-        tries: 60,
-        sleep: 10
-      )
+      tries, sleep = 60, 10
```

Comment:
@tyler-ball I didn't see a description of ```acceptable_errors``` on that page, so I looked at the code and saw

    rescue *acceptable_errors
    end

which led me to believe that option would just swallow the exception and not finish (since the block to ```Retryable.retryable``` seems to need to raise an exception in order to exit?). That option could use some docs for sure on how it behaves.

If it does work, then that's great :)